notebooks/getting_started/part3_exploring_cohorts.ipynb

```diff
@@ -1,15 +1,5 @@
 {
```

Comment:
<p>I've now been waiting for ~5 minutes, and I still see the spinner in the itkWidgets widget... How long is this step expected to take? Or is something not working?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
After I re-ran the cell, I did see the below, which is different from the image included in the static notebook - I think this can be confusing. I also think it will be helpful to include some details how to use user interface to configure visualization (how to show slices, how to do window level).

<img width="964" alt="image" src="https://user-images.githubusercontent.com/313942/234399908-a3635c8b-7f71-4a36-bc7c-3b073e3cf0a9.png">

I also noticed that color map assigned to the voxel values when shown in 2D is the same as in 3D volume rendering, which I have to say is very confusing. It took me some time to understand why I see violet shades while trying to adjust w/l in the 2d crossection. This is a comment outside of this PR, but I think it would make sense to have grayscale color map assigned to the 2d view by default. Also, I did figure out how to switch to mouse w/l control, but did not succeed in trying to adjust the w/l to my liking :-( 

@bnmajor I made a screen recording to demonstrate my attempts: https://app.screencast.com/wGqulZMcTrZtm. For the sake of comparison, here's how this works in Slicer - a more meaningful initial value for w/l, different sensitivity to the adjustment, and (of course, I am a lot more experienced with Slicer than itkWidgets, but anyway) much shorter time to a nice visualiation: https://app.screencast.com/1GlQnfWvGtVtv. Maybe there are some settings that can be tweaked to improve experience? Also, it looks like itkWidgets does multi-slice projection instead of rendering a single slice. Is that right?

Reply:
<p>Thank you so much for the feedback and screencasts @fedorov!</p><p> </p><blockquote>I've now been waiting for ~5 minutes, and I still see the spinner in the itkWidgets widget... How long is this step expected to take? Or is something not working?</blockquote><p>That is certainly a case of something not working! It's hard to know what though without any errors. I have noticed that these examples take longer than others I have been testing recently and I will need to do a bit more digging to determine if this is related to the data itself or to Colab (which is typically slower for me in general).</p><p> </p><blockquote>After I re-ran the cell, I did see the below, which is different from the image included in the static notebook - I think this can be confusing.</blockquote><p>I will confirm that it is always the same series chosen to prevent possible confusion. In this same vein - the screenshot is a z-slice because the very last cell run updates the viewer to show a slice. I can either a) use the screenshot of the 3D image instead so that it matches the initial visualization or b) separate the two into their own viewers so there will be two different screenshots. Whichever you would prefer.</p><p> </p><blockquote>I also think it will be helpful to include some details how to use user interface to configure visualization (how to show slices, how to do window level).</blockquote><p>I can add some examples of these steps!</p><p> </p><blockquote>This is a comment outside of this PR, but I think it would make sense to have grayscale color map assigned to the 2d view by default. Also, I did figure out how to switch to mouse w/l control, but did not succeed in trying to adjust the w/l to my liking :-(</blockquote><p>I will certainly raise the idea of a different color map and see what we can do there! As for the w/l adjustment I really appreciate the feedback and example! This is very useful to me in trying to get this right - I will iterate a bit more and keep you in the loop with improvements!</p>

Reply:
@fedorov I've added a cell at the end that includes a little more information of the available tools as well as an annotated screenshot with some of the options they will most likely need (changing between slice and volume view, w/l interactor, animation, resolution scale).